### PR TITLE
TF-1704,1705,1706 Fix some bug on server CYRUS

### PR DESCRIPTION
**Pull request by dab246**
## Issues

- [x] #1704 
- [x] #1705 
- [x] #1706 

## Solution

- `#1704`:  Set default `MaxCallsInRequest = 1` 
-  `#1705`:  Auto create default folders `(Inbox/Sent/Trash/Drafts)` if missing with appropriate roles
- `#1706`: Check `from` property of email. If is null or empty then display default avatar otherwise display avatar by `hashCode` of email

## Resolved

https://github.com/linagora/tmail-flutter/assets/80730648/d0431027-23e3-4b27-914d-b1aeb1024f28

### CODE REVIEW

**Review by sherlockvn — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Stage 1 - Install dependencies and build the app
 # This matches the flutter version on our CI/CD pipeline on Github
-FROM cirrusci/flutter:3.7.5 AS build-env
+FROM debian AS build-env
+
+RUN apt-get update && \
+    apt-get install -y curl git wget unzip
+RUN git clone --depth=1 --branch=3.7.5 https://github.com/flutter/flutter.git /usr/local/flutter
+ENV PATH="/usr/local/flutter/bin:/usr/local/flutter/bin/cache/dart-sdk/bin:${PATH}"
```
Comment (hoangdat) [inline Dockerfile:8]: IMO, no need to change it, we not run it in M1 in production. Only change it for local development

**Comment by chibenwa** [comment]
This monday morning, could you please take time to rebase this work on master pkease @dab246 @hoangdat

**Comment by hoangdat** [comment]
- more investigate on #1705 : is success to create default folder with `role`

**Comment by hoangdat** [comment]
work with v0.10.3

**Pull request merged**